### DISCUSSION TITLE
PAG

### DISCUSSION THREAD

**Discussion by henlopping** [Tencent/libpag#945] [Q&A] [answered]
我使用PAGView和PPAGComposition通过addLayer的方式设置一张bitmap，但是图片没有居中展示，请问怎么解决啊？

**Comment by domchen** [comment, marked as answer]
一般是PAGComposition的尺寸设置问题。你可以把PAGComposition的尺寸创建为跟Bitmap一样的尺寸，再添加到PAGView里应该就行了。如果你的PAGComposition尺寸没法修改，那就手动设置图片图层的setMatrix来改变位置。需要自己计算一下居中需要的矩阵。
